scripts/prepare-release.sh

```diff
@@ -29,6 +29,11 @@ for arch in `ls -1 $BIN_DIR/`;do
     source_file=$BIN_DIR/$arch/odo$suffix
     target_file=$RELEASE_DIR/odo-$arch$suffix.gz
 
+    # Create a gzip of the binary
     echo "gzipping binary $source_file as $target_file"
     gzip --keep --to-stdout $source_file > $target_file
+
+    # Move binaries to the release directory as well
+    echo "moving binary $source_file to release directory"
+    mv source_file $RELEASE_DIR/odo-$arch$suffix
```

Comment:
you are creating filename again, that means that the logic how to construct filename is in two places.

how about removing `.gz` suffix from `target_file`
```
target_file=$RELEASE_DIR/odo-$arch$suffix
```
updating gzip command to include `.gz`
```
gzip --keep --to-stdout $source_file > $target_file.gz
```

and using the same variable again 
```
mv source_file $RELEASE_DIR/$target_file
```

Reply:
Is it safe to move it those files? 
I'm not sure but other scripts that are running after this might expect those binaries to be there.
It is just hunch, I haven't checked that.